source/includes/oai-pmh/_root.md

```diff
@@ -143,6 +187,39 @@ correct community identifier.
 [See example](https://zenodo.org/oai2d?verb=ListRecords&
 metadataPrefix=oai_datacite&set=user-cfa)
 
+
```

Comment:
I think all these sections could use a brief text under the title describing the case.